frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -15,6 +15,7 @@ import { setCourse } from "../../../actions/coursesActions";
 import { BsPlusLg } from "react-icons/bs";
 
 const { Title, Text } = Typography;
+const term = "2022-T2";
```

Comment:
I think it should probably be better if this variable is defined as a `REACT_APP_` env var or create a file in the root of the `src` folder (something like `config.js`) that exports this variable. Would be easier to know how to change the `term` value if another dev is not familiar on how to change the term later on. What do you think?